server/server.go

```diff
@@ -1010,6 +1016,30 @@ var healthzData = []byte(`{
   "status": "ok"
 }`)
 
+func (s *Server) GetSSLCertificate(*tls.ClientHelloInfo) (*tls.Certificate, error) {
```

Comment:
Could you write a test for this function in server/server_test.go ?

Reply:
Will do

Reply:
I've added a test. Can you recommend the best place to put the self signed certificates if they aren't in a good spot.

Reply:
> I've added a test

Thank you for adding the test.

> Can you recommend the best place to put the self signed certificates if they aren't in a good spot.

I'm surprised there isn't a location already. However, I'm OK with putting it in the `./testdata/` dir for now until there is a better location.